scripts/embedded-pg/main.go

```diff
@@ -0,0 +1,65 @@
+// Start an embedded postgres database on port 5432. Used in CI on macOS and Windows.
+package main
+
+import (
+	"database/sql"
+	"os"
+	"path/filepath"
+
+	embeddedpostgres "github.com/fergusstrange/embedded-postgres"
+)
+
+func main() {
+	postgresPath := filepath.Join(os.TempDir(), "coder-test-postgres")
+	ep := embeddedpostgres.NewDatabase(
+		embeddedpostgres.DefaultConfig().
+			Version(embeddedpostgres.V16).
+			BinariesPath(filepath.Join(postgresPath, "bin")).
+			DataPath(filepath.Join(postgresPath, "data")).
+			RuntimePath(filepath.Join(postgresPath, "runtime")).
+			CachePath(filepath.Join(postgresPath, "cache")).
+			Username("postgres").
+			Password("postgres").
+			Database("postgres").
+			Encoding("UTF8").
+			Port(uint32(5432)).
+			Logger(os.Stdout),
+	)
+	err := ep.Start()
+	if err != nil {
+		panic(err)
+	}
+	// We execute these queries instead of using the embeddedpostgres
+	// StartParams because it doesn't work on Windows. The library
+	// seems to have a bug where it sends malformed parameters to
+	// pg_ctl. It encloses each parameter in single quotes, which
+	// Windows can't handle.
```

Comment:
Can we open a PR upstream to fix this bug?

Reply:
Sure. I opened https://github.com/fergusstrange/embedded-postgres/issues/145 and I could submit a PR later.

Reply:
Can you add the issue tracking link to this comment?